IPython/nbconvert/utils/lexers.py

```diff
@@ -49,10 +49,14 @@
 line_re = re.compile('.*?\n')
 
 ipython_tokens = [
-  (r'(\%+)(\w+)\s+(\.*)(\n)', bygroups(Operator, Keyword,
+  (r'(%)(sx|sc|system)(.*)(\n)', bygroups(Operator, Keyword,
                                        using(BashLexer), Text)),
-  (r'(\%+)(\w+)\b', bygroups(Operator, Keyword)),
```

Comment:
> These lexers are not only used in highlighting notebooks.

Please take a look at #7558.  I will make it much easier to test (and use) ipython lexers if they are exposed as pygments entry points.  (For example, pygments comes with a testcase formatter that converts plain code examples to unittests.)